src/api/app/controllers/webui/user_controller.rb

```diff
@@ -88,7 +88,7 @@ def save_dialog
   end
 
   def icon
-    size = params[:size].to_i || '20'
+    size = params[:size].to_i
```

Comment:
Did you check if we use this parameter anywhere? I grep'ed through the code and couldn't find any trace of it.

Reply:
This parameter is only used in the `user_icon` helper, only.

Reply:
because it's an action?

Reply:
@coolo who is using this action except us.

And given that it's used by a helper, it seems to be a helper itself.

Reply:
Alright, it's [routed](https://github.com/openSUSE/open-build-service/blob/master/src/api/config/routes.rb#L372) from user_icon to icon.

Reply:
well, it renders an icon